Horrible hack needed to fix FTBFS due to current headers not defining
NPNVSupportsWindowless.

Index: gecko-mediaplayer-0.9.6/src/plugin_setup.cpp
===================================================================
--- gecko-mediaplayer-0.9.6.orig/src/plugin_setup.cpp	2009-06-07 10:49:40.000000000 +0200
+++ gecko-mediaplayer-0.9.6/src/plugin_setup.cpp	2009-06-07 10:50:10.000000000 +0200
@@ -362,7 +362,7 @@
 
     }
 
-    nperror = NPN_GetValue (instance->mInstance, NPNVSupportsWindowless, &supportsWindowless);
+    nperror = NPN_GetValue (instance->mInstance, (NPNVariable)17, &supportsWindowless);
 	supportsWindowless = (nperror == NPERR_NO_ERROR) && supportsWindowless;
 	if (instance->windowless) {
 		if (supportsWindowless) {
